tests/42tags.pl

```diff
@@ -378,3 +382,49 @@ sub check_private_user_data {
          Future->done( 1 );
       });
    };
+
+test "Deleted tags appear in an incremental v2 /sync",
```

Comment:
Actually this appears not to be necessary; the test currently passes against synapse from `develop` (v0.10.1-rc1-208-g466b4ec)